web/source/dom/domEventHandlers.ts

```diff
@@ -256,7 +256,11 @@ namespace com.keyman.dom {
       }
 
       this.doChangeEvent(Ltarg);
-      this.keyman.textProcessor.keyboardInterface.resetContext();
+      let outputTarget = text.Processor.getOutputTarget(Ltarg as HTMLElement);
```

Comment:
This looks like we are introducing a new pattern of two `resetContext()` -- is it possible to wrap this in a function that does both calls (and document accordingly) -- see also below.